base/locks.jl

```diff
@@ -121,6 +159,16 @@ end
 
 const UV_MUTEX_SIZE = ccall(:jl_sizeof_uv_mutex, Cint, ())
 
+"""
+    Mutex()
+
+These are standard system mutexes for locking critical sections of logic.
+
+On Windows, this is a `critical section object`,
```

Comment:
why is `critical section object` highlighted as code? use its win32 name or don't highlight it